src/platform/forms-system/src/js/utilities/ui/index.js

```diff
@@ -49,7 +50,12 @@ export function scrollToFirstError() {
       document.body.scrollTop ||
       0;
     const position = errorEl.getBoundingClientRect().top + currentPosition;
-    Scroll.animateScroll.scrollTo(position - 10, getScrollOptions());
+    // Don't animate the scrolling if there is an open modal on the page. This
+    // prevents the page behind the modal from scrolling if there is an error in
+    // modal's form.
+    if (!body.classList.contains('modal-open')) {
```

Comment:
I'm worried about the reliability of this because adding a `modal-open` class to the body seems like a side-effect that might change at some point.

What do you think about checking to see if any element with the class `va-modal` appears on the page?  That still leaves the possibility for the classname to change so it may not be much better.  The only way around that would be if the modal updated some global state value each time it was opened and we could check that, but that seems complicated.

Reply:
Very true.

> The only way around that would be if the modal updated some global state value each time it was opened and we could check that, but that seems complicated.

Seems like the state the modal would change could change, too :-D

I'm not sure of the best approach here. But it might be helpful to at least add a comment in the code that toggles the `.modal-open` class, making it clear that this fix depends on that class existing on the `body`. I'll think about this for a bit and follow up.

Reply:
I don't think there's a bulletproof alternative to this. So I added a comment here https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/214. Could you give this an approval if this works for you? Thanks, @bkjohnson